src/app/child-dev-project/children/dashboard-widgets/birthday-dashboard/birthday-dashboard.component.ts

```diff
@@ -17,40 +18,47 @@ export class BirthdayDashboardComponent
 {
   @ViewChild(MatPaginator) paginator: MatPaginator;
   private readonly today: Date;
-  readonly birthdayThreshold = 32;
-  childrenDataSource = new MatTableDataSource<{
-    child: Child;
-    birthday: Date;
-  }>();
+  private entities = { [Child.ENTITY_TYPE]: "dateOfBirth" };
+  birthdayThreshold = 32;
+  dataSource = new MatTableDataSource<EntityWithBirthday>();
   isLoading = true;
 
   constructor(private entityMapper: EntityMapperService) {
     this.today = new Date();
     this.today.setHours(0, 0, 0, 0);
   }
 
-  async ngOnInit() {
-    const children = await this.entityMapper.loadType(Child);
-
-    // Only active children with a birthday in less than 31 days are shown
-    this.childrenDataSource.data = children
-      .filter((child: Child) => child.isActive && child.dateOfBirth)
-      .map((child) => ({
-        child: child,
-        birthday: this.getNextBirthday(child.dateOfBirth),
-      }))
-      .filter((a) => this.daysUntil(a.birthday) < this.birthdayThreshold)
-      .sort((a, b) => this.daysUntil(a.birthday) - this.daysUntil(b.birthday));
+  onInitFromDynamicConfig(config: BirthdayDashboardConfig) {
+    this.entities = config?.entities ?? this.entities;
+    this.birthdayThreshold = config?.threshold ?? this.birthdayThreshold;
+  }
 
+  async ngOnInit() {
+    const data: EntityWithBirthday[] = [];
+    for (const [entityType, property] of Object.entries(this.entities)) {
+      const entities = await this.entityMapper.loadType(entityType);
+      data.push(
+        ...entities
+          .filter((entity) => entity.isActive && entity[property])
+          .map((entity) => ({
+            entity: entity,
+            birthday: this.getNextBirthday(entity[property]),
+            age: entity[property]?.age + 1,
```

Comment:
suggestions `age` -> `newAge`? (I was wondering why we don't add +1 in the template anymore)